ndbench-geode-plugins/src/test/java/com/netflix/ndbench/geode/plugin/GeodeCloudPluginTest.java

```diff
@@ -0,0 +1,18 @@
+package com.netflix.ndbench.geode.plugin;
+
+import org.junit.Test;
+
+
+public class GeodeCloudPluginTest {
+
+  @Test
+  public void invalidVcapServicesVariableTest(){
+
```

Comment:
Can you write some tests for these plugins? I know we are lagging behind in test coverage for this project, but trying to stop the bleeding.